lib/tools.js

```diff
@@ -394,10 +395,16 @@ function createArgsNormalizer (defaultOptions) {
       stream = buildSafeSonicBoom({ dest: opts, sync: true })
       opts = {}
     } else if (typeof stream === 'string') {
+      if (opts && opts.transport) {
+        throw Error('Only one of option.transport or stream can be specified')
```

Comment:
Are we throwing errors with uppercase strings anywhere else? If not, we should be consistent by:

```suggestion
        throw Error('only one of option.transport or stream can be specified')
```